### PR TITLE
New AddHeader() to specify arbitrary headers

### PR DESCRIPTION
**Pull request by deanishe**
As discussed in #26

### CODE REVIEW

File: examples_test.go [hunk 1]
```diff
@@ -19,6 +19,9 @@ func Example() {
 
 	mail.Subject("Business proposition")
 
+	// Add a custom header
+	mail.AddHeader("X-TOTALLY-NOT-A-SCAM", "true")
```
Comment (domodwyer) [inline examples_test.go]: Love it!

**Comment by domodwyer** [comment]
Thanks very much @deanishe 

Sorry for the delay - started a new job so it's been a busy week!

Thanks again,
Dom

**Comment by deanishe** [comment]
No worries. Hope you're getting along well at the new place!

File: examples_test.go (hunk 1 above)
Comment (deanishe) [reply examples_test.go]: 😁 It seemed appropriate to the example.

**Pull request merged**